packages/plugins/src/cli/utils.ts

```diff
@@ -27,3 +29,11 @@ export const coerceNetworks = (
   // If none of the above conditions are met, throw a general error.
   throw new Error(getInvalidNetworksArgumentError(arg))
 }
+
+export const checkLibraryVersion = (libraryVersion: string) => {
+  if (libraryVersion !== CONTRACTS_LIBRARY_COMMIT_HASH) {
+    throw Error(
+      'Incorrect Sphinx library contracts version detected, please run `npx sphinx install` to update to the required version.'
```

Comment:
nit: This sentence is awkwardly constructed. I think there should be a period after "detected", i.e.
> Incorrect ... version detected. Please run ...